tests/unit/pom.xml

```diff
@@ -55,6 +55,12 @@
             <artifactId>mule-tests-model</artifactId>
             <version>${project.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.mule.services</groupId>
+            <artifactId>mule-service-scheduler</artifactId>
```

Comment:
remove this dependency. Using this would make the test be less unit and more integration